lib/linux_admin/process/memory.rb

```diff
@@ -0,0 +1,49 @@
+module LinuxAdmin
+  module Process
+    class Memory
```

Comment:
Wondering if this should be part of the Process class directly?  Just thinking about how we plan to transfer a lot of the miq-process stuff over to here.  @carbonin @bdunne Thoughts on the class layout?